zenoh/tests/acl.rs

```diff
@@ -1748,3 +1738,200 @@ async fn test_liveliness_deny_allow_query(port: u16) {
     close_sessions(reader_session, writer_session).await;
     close_router_session(session).await;
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+async fn test_acl_query_ingress_deny() {
+    zenoh::init_log_from_env_or("error");
+    let mut config_router = get_basic_router_config(27501).await;
+    config_router
+        .insert_json5(
+            "access_control",
+            r#"{
+            "enabled": true,
+            "default_permission": "allow",
+            "rules": [
+                {
+                    "id": "deny query ingress",
+                    "permission": "deny",
+                    "messages": ["query", "liveliness_query"],
```

Comment:
`"liveliness_query"` is not related to this test
```suggestion
                    "messages": ["query"],
```

Reply:
fixed